wicket-kendo-ui-culture/src/main/java/com/googlecode/wicket/kendo/ui/KendoCultureHeaderContributor.java

```diff
@@ -0,0 +1,47 @@
+package com.googlecode.wicket.kendo.ui;
+
+import java.util.Locale;
+
+import org.apache.wicket.markup.head.IHeaderResponse;
+import org.apache.wicket.markup.html.IHeaderContributor;
+
+/**
+ * IHeaderContributor to easily add {@link KendoCultureHeaderItem} to each page by using
+ * 
+ * <pre>
+ * org.apache.wicket.Application#getHeaderContributorListeners().add(new KendoCultureHeaderContributor())
+ * </pre>
+ * Usage:
+ *
+ * <pre>
+ * <code>
+ * public class MyApplication extends WebApplication
+ * {
+ * 	public void init()
+ * 	{
+ * 		super.init();
+ * 
+ * 		getHeaderContributorListenerCollection().add(new KendoCultureHeaderContributor());
```

Comment:
Ah, missed to update the javadoc. Can I just do another push again and it will be part of this pull request?